rep-I0001.rst

```diff
@@ -185,26 +200,11 @@ Node Configuration
 ---------
 Similar to the motion interface, the state interface will require configuration.  The state interface will have to parse messages coming from the robot and convert the date into the desired ROS topics.  The level of configuration available on the robot controller will vary, so the messages coming from the controller may be more or less dynamic.  The state node, based on configuration, will identify the pertinent information from the robot controller and convert to ROS topics.  Additional information will be ignored.  
 
-The yaml file will contain a list of structures that defines the joint trajectory/status topics as well as the mapping to the controller.  Note, this configuration is very similar to the motion node, with the exception that the state node performs a one-to-one mapping from controller groups to topics.  The motion node, in addition to this, can perform a one(topic) to many (groups) mapping.::
-
-        topic_list:
-         - state
-           group: <controller group#>
-            - joint
-             - name: <topic name>
-              ns: <topic namespace>
-               joints:
-                - <joint_1>
-                - <joint_2>
-                - <joint_N>
-         - status
-           - name: <topic name>
-           - ns: <topic namespace>
-           ...
+The state interface uses the same ROS configuration parameter as the motion node.  Robot groups may be mapped to the same namespace (aggregate status messages) or different namespaces (per-group messages), or both.  As in the motion interface, the same topics are used within each namespace as are used in the current state interface.
```

Comment:
Perhaps add some example configurations for each of the cases? The "may be mapped to X, Y or both" can be confusing.